en/computer_vision/collision_prevention.md

```diff
@@ -1,89 +1,101 @@
 # Collision Prevention
 
 _Collision Prevention_ may be used to automatically slow and stop a vehicle before it can crash into an obstacle.
+It can be enabled for multicopter vehicles when using acceleration-based [Position mode](../flight_modes_mc/position.md).
```

Comment:
```suggestion
It can be enabled for multicopter vehicles when using acceleration-based [Position mode](../flight_modes_mc/position.md) (or VTOL vehicles in MC mode).
```